AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Core/DataStoreList.swift

```diff
@@ -55,12 +61,32 @@ public class DataStoreList<ModelType: Model>: List<ModelType>, ModelListDecoder
 
     // MARK: - Collection conformance
 
+    /// This method implicitly lazy loads the underlying collection so the runtime of using Collection methods that
+    /// are normally O(1) are decreased to the time it takes to retrieve the data from the underlying SQL data source
+    /// for the initial method call. After the first access, the data will be retrieved and subsequent calls will be
+    /// O(1) runtime.
```

Comment:
suggested reword here and throughout

```suggestion
    /// for the initial method call. After the first access the data is already retrieved, so subsequent calls will be
    /// performed in O(1).
```